e2e/tests/strategy/createLimit.ts

```diff
@@ -0,0 +1,75 @@
+import { expect, test } from '@playwright/test';
+import { CreateStrategyTemplate } from './../../utils/strategy/template';
+import { checkApproval } from './../../utils/modal';
+import { NotificationDriver } from './../../utils/NotificationDriver';
+import {
+  fiatPrice,
+  navigateTo,
+  tokenPrice,
+  waitFor,
+} from './../../utils/operators';
+import { CreateStrategyDriver, MyStrategyDriver } from './../../utils/strategy';
+
+export const createLimitStrategy = (config: CreateStrategyTemplate) => {
+  const { base, quote, buy, sell } = config;
+  const buyBudgetFiat = parseFloat(buy.budgetFiat ?? '0');
+  const sellBudgetFiat = parseFloat(sell.budgetFiat ?? '0');
+
+  return test(`Create`, async ({ page }) => {
+    test.setTimeout(45_000);
+
+    await waitFor(page, `balance-${quote}`, 30_000);
+
+    await navigateTo(page, '/');
+    const myStrategies = new MyStrategyDriver(page);
+    const createForm = new CreateStrategyDriver(page, config);
+    await myStrategies.createStrategy();
+    await createForm.selectToken('base');
+    await createForm.selectToken('quote');
+    await createForm.selectSetting('two-ranges');
+    await createForm.nextStep();
+    const buyForm = await createForm.fillLimit('buy');
+    const sellForm = await createForm.fillLimit('sell');
+
+    // Assert 100% outcome
+    await expect(buyForm.outcomeValue()).toHaveText(`0.006666 ${base}`);
```

Comment:
Maybe add a todo so we remember to remove hardcoded value : 
```suggestion
    // TODO: derive value from config instead of hardcoding it
    await expect(buyForm.outcomeValue()).toHaveText(`0.006666 ${base}`);
```